### PR TITLE
Count with Prototype (Ajax) removed + do not test mingw package on 5.4

### PR DESCRIPTION
**Pull request by mfalesni**
The Prototype framework was removed completely so the script needs to count with the fact that Ajax can be undefined. Then it returns just 0 if Ajax is not defined, making everything backwards-compatible

{{pytest: -m smoke -v --use-provider vsphere55}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 30cff401074e06f1be52f4a315294e7c567fb842:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/js.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 return {
     jquery: jQuery.active,
-    prototype: Ajax.activeRequestCount,
+    prototype: (typeof Ajax == "undefined") ? 0 : Ajax.activeRequestCount,
```
Comment (seandst) [inline cfme/js.py]: I'd rather avoid the potential for weird JS bugs, so using the no-really-are-they-equal operator seems prudent (`===` vs. `==`).

Comment (mfalesni) [inline cfme/js.py]: Are you sore you want to be so paranoid? typeof returns string, undefined is a string so this should not make troubles ...

**Comment by dajoRH** [comment]
Lint report for commit 34a7d859bc936100978132f5b5748d4d3841fe0d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/js.py (hunk 1 above)
Comment (seandst) [inline cfme/js.py]: Yup. Unless you explicitly want the `==` behavior instead of `===`, `===` should always be preferred.

Comment (mfalesni) [inline cfme/js.py]: OK then.

**Comment by dajoRH** [comment]
Lint report for commit 43653afbbf647923cc0729d4c444458878a05945:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**